requirements.txt

```diff
@@ -0,0 +1,12 @@
+beautifulsoup4==4.9.3
+certifi==2020.12.5
+chardet==4.0.0
+flake8==3.8.4
+idna==2.10
+lxml==4.6.2
+mccabe==0.6.1
+pycodestyle==2.6.0
+pyflakes==2.2.0
+requests==2.25.1
+soupsieve==2.1
+urllib3==1.26.2
```

Comment:
nice usage of requirements.txt, this made it very easy to get this program's required modules installed